README.md

```diff
@@ -123,6 +133,43 @@ The simplest usage of this task is to make a GET request by only setting the
   return: $request
   ```
 
+### Making a JSON request
+
+If you are making a request to a JSON endpoint, you can configure the task to
+automatically convert the body to JSON, set the request headers, and parse
+the response body as JSON. To enable this behavior, set the `json_endpoint`
+parameter to `true`.
+
+- **Unix-like shell command**
+
+  ```shell
+  $ bolt task run http_request --targets localhost base_url=http://httpbin.org/get json_endpoint=true
+  ```
+
+- **PowerShell command**
+
+  ```powershell
+  > Invoke-BoltTask -Name 'http_request' -Targets 'localhost' base_url=http://httpbin.org/get json_endpoint=true
+  ```
+
+- **Plan step**
+
+  ```yaml
+  parameters:
+    targets:
+      type: TargetSpec
+
+  steps:
+    - name: request
+      task: http_request
+      targets: $targets
+      parameters:
+        base_url: 'http://httpbin.org/get'
+        json_endpoint: true
+
+  return: $request
```

Comment:
```suggestion
  return: $request.first.value['body']['headers']['User-Agent']
```
Or something, just to demonstrate accessing a value from the result

Reply:
Er sorry I don't think this got updated?

Reply:
Only updated below?

Reply:
D'oh.